modules/extensions-spring-support/src/test/java/org/mule/module/extension/internal/capability/xml/SchemaGeneratorTestCase.java

```diff
@@ -38,7 +37,7 @@
     public void before()
     {
         generator = new SchemaGenerator();
-        extensionFactory = new DefaultExtensionFactory(mock(ServiceRegistry.class));
+        extensionFactory = new DefaultExtensionFactory(new SpiServiceRegistry(), getClass().getClassLoader());
```

Comment:
I can, but instead of adding a when I think there's more value in actually testing that the SPI thing is working and picking up the enricher